### PR TITLE
Update CSES Flight Discount.mdx

### PR DESCRIPTION
**Pull request by KWang31**
Added formatted explanation

### CODE REVIEW

**Comment by dolphingarlic** [comment]
Could you flesh out the editorial a bit please? It's very bare-bones right now (also please use LaTeX for all variables like N)

**Comment by KWang31** [comment]
This is a meet in the middle problem. Take the obvious graph interpretation.

Say I use the discount on edge between A and B, and I use the edge.

There are two cases: I have to go from $1\rightarrow A\rightarrow B\rightarrow N$, or $1\rightarrow B\rightarrow A\rightarrow N$. It turns out that we need the distance between 1 and A, and N and B.

This means we use dijkstra to compute the distance from 1, N to every vertex, then compute $\text{min}_{A\rightarrow B} dist1[A]+c(A,B)+distN[B]$ where $c(A,B)$ is the cost to travel from city A to city B.

File: solutions/CSES Flight Discount.mdx [hunk 1]
```diff
@@ -3,8 +3,15 @@ id: cses-1195
...
 author: Kai Wang
 ---
+This is a meet in the middle problem. Take the obvious graph interpretation.
 
-<IncompleteSection />
+Say I use the discount on edge between A and B, and I use the edge.
+
+There are two cases: I have to go from $1\rightarrow A\rightarrow B\rightarrow N$, or $1\rightarrow B\rightarrow A\rightarrow N$. It turns out that we need the distance between 1 and A, and N and B.
+
+This means we use dijkstra to compute the distance from 1, N to every vertex, then compute $\text{min}_{A\rightarrow B} dist1[A]+c(A,B)+distN[B]$,
```
Comment (thecodingwizard) [inline solutions/CSES Flight Discount.mdx:12]:
```suggestion
We can use Dijkstra's to compute the distance from $1$ and $N$ to every vertex. Then our answer is $\text{min}_{A\rightarrow B} dist1[A]+c(A,B)+distN[B]$,
```

File: solutions/CSES Flight Discount.mdx [hunk 2]
```diff
@@ -3,8 +3,15 @@ id: cses-1195
...
+
+where $c(A,B)$ is the cost to travel from city A to city B.
```
Comment (thecodingwizard) [inline solutions/CSES Flight Discount.mdx:14]:
```suggestion
where $c(A,B)$ is the cost to travel from city A to city B after applying the coupon to that flight.
```

**Review by thecodingwizard — CHANGES_REQUESTED** [review]
looks good, just some minor wording changes that might make the solution more clear?

File: solutions/CSES Flight Discount.mdx [hunk 3]
```diff
@@ -3,8 +3,15 @@ id: cses-1195
 title: CSES - Flight Discount
 author: Kai Wang
 ---
+This is a meet in the middle problem. Take the obvious graph interpretation.
 
-<IncompleteSection />
+Say I use the discount on edge between A and B, and I use the edge.
```
[the remark this answers is not in the document]
Comment (KWang31) [reply solutions/CSES Flight Discount.mdx:8]: Yes that is clearer.

File: solutions/CSES Flight Discount.mdx [hunk 4]
```diff
@@ -3,8 +3,15 @@ id: cses-1195
 title: CSES - Flight Discount
 author: Kai Wang
 ---
+This is a meet in the middle problem. Take the obvious graph interpretation.
```
[the remark this answers is not in the document]
Comment (KWang31) [reply solutions/CSES Flight Discount.mdx:6]: Good point and I don't know... Here is supporting evidence from geeksforgeeks.com: "Like divide and conquer it splits the problem into two, solves them individually and then merge them"

File: solutions/CSES Flight Discount.mdx (hunk 1 above)
Comment (KWang31) [reply solutions/CSES Flight Discount.mdx:12]: Yes

File: solutions/CSES Flight Discount.mdx (hunk 2 above)
Comment (KWang31) [reply solutions/CSES Flight Discount.mdx:14]: Oh yes that was a typo oops

File: solutions/CSES Flight Discount.mdx [hunk 5]
```diff
@@ -3,8 +3,15 @@ id: cses-1195
...
 
-<IncompleteSection />
+Say I use the discount on edge between A and B, and I use the edge.
+
+There are two cases: I have to go from $1\rightarrow A\rightarrow B\rightarrow N$, or $1\rightarrow B\rightarrow A\rightarrow N$. It turns out that we need the distance between 1 and A, and N and B.
```
[the remark this answers is not in the document]
Comment (KWang31) [reply solutions/CSES Flight Discount.mdx:10]: yes

**Comment by thecodingwizard** [comment]
@KWang31 if the changes I made look good to you, I'll merge this PR. Thanks!

**Review by thecodingwizard — APPROVED** [review]

File: solutions/CSES Flight Discount.mdx (hunk 4 above)
[the remark this answers is not in the document]
Comment (KWang31) [reply solutions/CSES Flight Discount.mdx]: Maybe delete "take the obvious graph interpretation..." it's too formal and unnecessary.

[the remark this answers is not in the document]
Comment (thecodingwizard) [reply solutions/CSES Flight Discount.mdx]: I think both sentences are deleted...

**Comment by KWang31** [comment]
The changes looks good to me. I will pay more attention to my wording when writing future editorials.

**Comment by thecodingwizard** [comment]
Awesome, thanks for your contributions!

**Pull request merged**